### ISSUE TITLE
SmartException: At least one shipping rate computation method provider is required to be active.

### ISSUE THREAD

**Issue by mchinchilla** [smartstore/Smartstore#482]
An unhandled exception occurred while processing the request.
SmartException: At least one shipping rate computation method provider is required to be active.

Smartstore.Core.Checkout.Shipping.ShippingService.LoadActiveShippingRateComputationMethods(int storeId, string systemName) in ShippingService.cs, line 77

**To Reproduce**
![image](https://user-images.githubusercontent.com/5103578/156497104-29002ff2-f12a-4b90-a55f-8b82f1671190.png)

![image](https://user-images.githubusercontent.com/5103578/156497119-651f8778-7ded-4a2a-8107-9d85728dc3df.png)

**Comment by Michael-Herzog** [comment]
Perhaps the shipping module wasn't compiled. PLease check this here:
\src\Smartstore.Web\Modules

**Comment by mchinchilla** [comment]
Yes, the module is there (debug and published), but the same result.
![image](https://user-images.githubusercontent.com/5103578/159102313-497d2488-dde5-452f-8125-5fac31509fb9.png)
How can I check if the module was loaded?

**Comment by Michael-Herzog** [comment]
Please have a look here
Admin > Plugins > Manage Plugins

**Comment by Michael-Herzog** [comment]
How did you solve this problem?

**Comment by mchinchilla** [comment]
I installed the plugin: Admin > Plugins > Manage Plugins.

But I had an error with the migration, I reinstalled the database and now it's working.
